src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -42,7 +42,8 @@ trait MethodConfigurationService extends HttpService with PerRequestCreator with
               val extReq = Post(FireCloudConfig.Rawls.copyFromMethodRepoConfigUrl, copyMethodConfig)
               externalHttpPerRequest(requestContext, extReq)
             }
-          }
+          } ~
+          complete(StatusCodes.MethodNotAllowed, ErrorReport(StatusCodes.MethodNotAllowed, "HTTP method not allowed"))
```

Comment:
Right, the only purpose is to wrap it in an ErrorReport.

I really like your default RejectionHandler idea.  I'll look into that.  Do you know of any convenient examples?